compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -223,6 +225,11 @@ class ModelChef
   std::vector<flatbuffers::Offset<::tflite::Operator>> _operator_vec;
 
   std::string _graph_name;
+
+  // store Buffer data to external of FB and use (Buffer) offset/size fields
+  bool _ext_offset = false;
```

Comment:
There is nowhere to set this flag to `true` yet, right?

Reply:
yup, will be added in another PR.